18/README.md

```diff
@@ -0,0 +1,208 @@
+        "Frank's node id",
+	"Frank's hostname",
+	"New voting power"
+      ]
+    }
+  },
+  "id": "e...047",
+  "inputs": [
+    {
+      "fulfillment": "p...34C",
+      "fulfills": null,
+      "owners_before": [
+        "Alice's public key"
+      ]
+    }
+  ],
+  "metadata": null,
+  "operation": "CREATE",
+  "outputs": [
+    {
+      "amount": "1",
+      "condition": {
+        "details": {
+          "public_key": "Alice's public key",
+          "type": "ed25519-sha-256"
+        },
+        "uri": "ni:///sha-256;c...7-8Y?fpt=ed25519-sha-256&cost=131072"
+      },
+      "public_keys": [
+        "Alice's public key"
+      ]
+    },
+    {
+      "amount": "1",
+      "condition": {
+        "details": {
+          "public_key": "Bob's public key",
+          "type": "ed25519-sha-256"
+        },
+        "uri": "ni:///sha-256;b...123?fpt=ed25519-sha-256&cost=131072"
+      },
+      "public_keys": [
+        "Bob's public key"
+      ]
+    },
+    {
+      "amount": "1",
+      "condition": {
+        "details": {
+          "public_key": "Carly's public key",
+          "type": "ed25519-sha-256"
+        },
+        "uri": "ni:///sha-256;c...13C?fpt=ed25519-sha-256&cost=131072"
+      },
+      "public_keys": [
+        "Carly's public key"
+      ]
+    },
+    {
+      "amount": "1",
+      "condition": {
+        "details": {
+          "public_key": "Daniel's public key",
+          "type": "ed25519-sha-256"
+        },
+        "uri": "ni:///sha-256;c...7-8Y?fpt=ed25519-sha-256&cost=131072"
+      },
+      "public_keys": [
+        "Daniel's public key"
+      ]
+    }
+  ],
+  "version": "2.0"
+}
+```
+
+Now that the Election has been created, each Member can cast their vote. Bob starts:
+```json
+{
+  "asset": {
+    "id": "e...047"
+  },
+  "id": "a...123",
+  "inputs": [
+    {
+      "fulfillment": "f...qE8p",
+      "fulfills": {
+        "output_index": 1,
+        "transaction_id": "e...047"
+      },
+      "owners_before": [
+        "Bob's public key"
+      ]
+    }
+  ],
+  "metadata": {
+    "type": "vote"
+  },
+  "operation": "TRANSFER",
+  "outputs": [
+    {
+      "amount": "1",
+      "condition": {
+        "details": {
+          "public_key": "Election Address",
+          "type": "ed25519-sha-256"
+        },
+        "uri": "ni:///sha-256;b...123?fpt=ed25519-sha-256&cost=131072"
+      },
+      "public_keys": [
+        "Election Address"
+      ]
+    }
+  ]
+  "version": "2.0"
+}
+```
+
+## Rationale
+<!--The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
+Another idea to implement this protocol is to use a multi signature transaction.
```

Comment:
The Rationale section should explain why the above design was chosen rather than some other design. In this case, you should explain why a multi-signature transaction wasn't used.

Reply:
Yes, I'll add it soon.